package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "formidable",
-  "version": "3.0.0-canary.20210428",
+  "version": "3.1.0-canary.20210914",
```

Comment:
think minor should remain `0`

Reply:
I upped minor to publish on my personal npm (semver) since I had already 3.0.0

Reply:
Alrighty :)